scripts/secnetperf-helpers.psm1

```diff
@@ -483,6 +489,9 @@ function Invoke-Secnetperf {
         continue
     }
 
+     # Linux XDP requires env var and sudo for now
+    $sudo = (!$IsWindows -and $io -eq "xdp") ? "sudo MSQUIC_ENABLE_XDP=1 " : ""
```

Comment:
Daiki, you had `sudo -E` here, but I replaced it with the above because I thought that was actually what you needed, right? Please confirm before merging.

Reply:
Oh, I see, you do it in the other script with `$env:`. I'll change back to `-E`